src/components/callout/examples/callout-styles.tsx

```diff
@@ -15,7 +15,7 @@ import { Component, h } from '@stencil/core';
 export class CalloutStylesExample {
     public render() {
         return [
-            <limel-callout type="caution">
+            <limel-callout heading="Example" type={'example' as any}>
```

Comment:
Yeah, I that's a good idea, I just did this because there was CSS for it in the example.
But I guess it's just old code that wasn't removed some time

Reply:
But I'm also a bit confused. Why would someone need to change the `type`? They can change the icon and heading anyways?

Reply:
Fixed

Reply:
Our scenario is that we want to add success and a failed/warning callout.
And for us to be able to set styles, we needed to set the type to something and then style it.
So for example set it to typ "tip" and then style that as a success. It was confusing.

So with this PR we can now do like this:
```html
<limel-callout type="success" icon="checked"/>
<limel-callout type="warning"/>
```

```css
limel-callout[type='success'] {
    --callout-color: rgb(var(--color-green-default));
    --callout-text-color: rgb(var(--color-green-dark));
}

limel-callout[type='warning'] {
    --callout-color: rgb(var(--color-red-default));
    --callout-text-color: rgb(var(--color-red-dark));
}
```

Which was much cleaner